components/schema-migrator/migrations/director/202003231306_add_automatic_scenario_assignement.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE automatic_scenario_assignments (
+    scenario VARCHAR(128),
+    tenant_id UUID NOT NULL,
+    key VARCHAR(256) NOT NULL,
+    value JSONB);
```

Comment:
I am wondering if the value should be of type JSONB, or just pure string. We discussed that we are going to support only simple values. @crabtree what do you think?

Reply:
I'd go with simple string here.